Tasks/TwineAuthenticateV1/make.json

```diff
@@ -0,0 +1,28 @@
+{
+    "common": [
+        {
+            "module": "../Common/utility-common",
```

Comment:
utility-common-v2 here too

Reply:
@zarenner and I were talking about this yesterday. The only thing we use utility-common is for telemetry. Zach was going to look for options to remove this dependency altogether. However, he did say that utility-common-v2 has some very specific code from some other team, so not sure if it's worth taking dependency on.

Reply:
Yeah, I just see that @zarenner is adding it to the new shared lib, that'll do the trick.